tests/sentry/incidents/test_action_handlers.py

```diff
@@ -150,7 +150,7 @@ def test_user(self):
             handler.resolve()
         out = mail.outbox[0]
         assert out.to == [self.user.email]
-        assert out.subject.startswith("Alert Rule Resolved")
+        assert out.subject == u"[Resolved] {} - {}".format(incident.title, self.project.slug)
```

Comment:
This fails with the status being `Open` instead of `Resolved`, any ideas?

Reply:
This test is kind of a hack, since we just fire a resolved email for an Incident that isn't resolved. If you update the incident status to resolved it should work.

Reply:
Makes sense, thanks!